src/sidebar/store/test/index-test.js

```diff
@@ -19,172 +20,206 @@ const fixtures = immutable({
 });
 
 describe('store', function() {
-  let store;
+  let fakeDefaultKeys;
+  let fakeLocalStorage;
   let fakeRootScope;
 
-  function tagForID(id) {
-    const storeAnn = store.findAnnotationByID(id);
-    if (!storeAnn) {
-      throw new Error(`No annotation with ID ${id}`);
-    }
-    return storeAnn.$tag;
-  }
-
   beforeEach(function() {
+    fakeDefaultKeys = {};
+    fakeLocalStorage = { getItem: sinon.stub() };
     fakeRootScope = { $applyAsync: sinon.stub() };
-    store = storeFactory(fakeRootScope, {});
+
+    $imports.$mock({
```

Comment:
This plain old doesn't seem to work here. This is where I'ma need some halp.

Reply:
The values of the object passed as the mocks for a particular module is a mapping of export names to values. In the case of an ES module with a default export which is an object, there is a single export name (`default`) and the value of that should be an object. See https://github.com/robertknight/babel-plugin-mockable-imports#mocking-default-exports.

CommonJS modules are conceptually different in that they only have a single `exports` value, rather than being able to have a `default` export as well as named exports. Some of our modules worked around that limitation by adding named properties to the default export.